OpenRA.Game/FileSystem/FileSystem.cs

```diff
@@ -322,6 +322,24 @@ public static string ResolveAssemblyPath(string path, Manifest manifest, Install
 			return File.Exists(resolvedPath) ? resolvedPath : null;
 		}
 
+		public static string ResolveCaseInsensitivePath(string path)
+		{
+			var resolved = Path.GetPathRoot(path);
+
+			if (resolved == null)
+				return null;
+
+			foreach (var name in path.Substring(resolved.Length).Split(Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar))
+			{
+				resolved = Directory.GetFileSystemEntries(resolved).FirstOrDefault(e => e.Equals(Path.Combine(resolved, name), StringComparison.InvariantCultureIgnoreCase));
```

Comment:
Adding the following before this line avoids the crash, but i'm not sure its the most correct fix:
```csharp
// Filter out paths of the form /foo/bar/./baz
if (name == ".")
	continue;
```

Reply:
This happens because of the `copy: .` definitions in the installer yaml, so I think its probably fine to do the stripping here.